executor/distsql.go

```diff
@@ -973,25 +973,26 @@ func (w *tableWorker) executeTask(ctx context.Context, task *lookupTableTask) er
 		sort.Sort(task)
 	}
 
-	if handleCnt != len(task.rows) {
-		if len(w.idxLookup.tblPlans) == 1 {
-			obtainedHandlesMap := make(map[int64]struct{}, len(task.rows))
-			for _, row := range task.rows {
-				handle := row.GetInt64(w.handleIdx)
-				obtainedHandlesMap[handle] = struct{}{}
-			}
-
-			logutil.Logger(ctx).Error("inconsistent index handles", zap.String("index", w.idxLookup.index.Name.O),
-				zap.Int("index_cnt", handleCnt), zap.Int("table_cnt", len(task.rows)),
-				zap.Int64s("missing_handles", GetLackHandles(task.handles, obtainedHandlesMap)),
-				zap.Int64s("total_handles", task.handles))
-
-			// table scan in double read can never has conditions according to convertToIndexScan.
-			// if this table scan has no condition, the number of rows it returns must equal to the length of handles.
-			return errors.Errorf("inconsistent index %s handle count %d isn't equal to value count %d",
-				w.idxLookup.index.Name.O, handleCnt, len(task.rows))
-		}
-	}
+	// Hot fix to ignore inconsistent error.
```

Comment:
maybe this will let admin check table cannot work.

maybe add  `w.checkIndexValue != nil && ` to this if?

Reply:
Great